sgt-back-end/index.js

```diff
@@ -0,0 +1,126 @@
+// eslint-disable-next-line no-console
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+
+  db.query(sql)
+    .then(result => {
+
+      const grades = result.rows;
+      if (!grades) {
+        res.status(404).json({
+          error: 'No Grades Recorded'
+        });
```

Comment:
`result.rows` is an array which is always truthy. also, an empty array is ok, not an error

Reply:
Okay, cool, I'll make the changes.